homework/08LessonEight/script.js

```diff
@@ -17,30 +17,31 @@ $(document).on('ready', function() {
 
   // Problem 1:
   // Get the children of #first-list using the .children() method
-  window.firstListChildren = null;
+  window.firstListChildren = $('#first-list').children();
```

Comment:
Problem 1 and 2 fail because you're not using the `$firstList` and `$secondList` variables provided for you, instead making two calls each on the same DOM objects.